content/release-0.24.0.md

```diff
@@ -1,3 +1,142 @@
 Title: Pandas Extension Arrays
 Date: 2019-01-04
 
+Extensibility has been of the major themes in pandas development over the last
+couple of releases. This post introduces the extension array interface: the
+motivation behind them and how they might affect you as a pandas user. Finally,
+we look at how extension arrays shape the future of pandas.
+
+## The Motivation
+
+Pandas is built on top of NumPy. You could roughly define a Series as a wrapper
+around a NumPy array, and a DataFrame as a collection of Series with a shared
+index. That's not entirely correct for several reasons, but I want to focus on
+the "wrapper around a NumPy array" part. It'd be more correct to say "wrapper
+around an array-like object".
+
+While pandas mostly used NumPy's type system, we restricted it in places and
```

Comment:
The term "type system" is one that seems to people seem to have different interpretations of. It might be more accurate to say "NumPy's built-in data representations" (where a dtype in NumPy has a 1-1 mapping onto a data representation)